doc/19_Development_Tools_and_Details/29_Testing/01_Application_Testing.md

```diff
@@ -355,7 +355,6 @@ suite to use the `\Pimcore\Tests\Support\Helper\Pimcore` helper:
 actor: FunctionalTester
 modules:
     enabled:
-        - \Tests\Support\Helper\Functional
```

Comment:
i believe it's a placeholder for example purposes, the entire FunctionalTester(`functional.suite.yml`) is not set or exists in the repository